camel/models/anthropic_model.py

```diff
@@ -108,7 +108,11 @@ def count_tokens_from_prompt(self, prompt: str) -> int:
         """
         return self.client.count_tokens(prompt)
 
-    @api_keys_required("ANTHROPIC_API_KEY")
+    @api_keys_required(
```

Comment:
I tested it and found that when the `api_keys_required` decorator is used in other methods, 
even if: `__init__(api_key="xxx"):` is explicitly set, errors will still be reported, because the current method does not have the parameter `api_key`, so it can't Verify `api_key` and just get the value from the environment variable.

I understand that this decorator hopes to be compatible with classes and independent methods, so I guess the current approach is only for method parameter verification and lacks support for classes?

Reply:
updated